### PR TITLE
Container updates and issues

### PR DESCRIPTION
**Pull request by uk-bolly**
**Overall Review of Changes:**
Container updates
lint
ansible version alignment

**Issue Fixes:**
#295 
#296 

**Enhancements:**
issues container some great improvements

**How has this been tested?:**
Manually with multiple options

### CODE REVIEW

File: vars/main.yml [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-min_ansible_version: 2.9
+min_ansible_version: 2.10.1
 rhel8cis_allowed_crypto_policies:
     - 'DEFAULT'
     - 'FUTURE'
     - 'FIPS'
 
+rhel8cis_allowed_crypto_policies_modules:
+    - 'OSPP'
+    - 'AD-Support'
```
Comment (bbaassssiiee) [inline vars/main.yml:12]: Should be all-caps

**Review by bbaassssiiee — CHANGES_REQUESTED** [review]
AD-SUPPORT is the name of the module.

**Comment by uk-bolly** [comment]
> AD-SUPPORT is the name of the module.

Great spot, updated

File: vars/main.yml (hunk 1 above)
Comment (uk-bolly) [reply vars/main.yml]: Great spot updated

**Review by MrSteve81 — APPROVED** [review]
Looks good to Me!

**Review by bbaassssiiee — APPROVED** [review]
I verified the fixes of my reported issues with the branch. Please merge and tag ASAP!

**Pull request merged**